app/app.go

```diff
@@ -2,9 +2,9 @@ package app
 
 import (
 	"fmt"
-	"github.com/SUSE/saptune/sap/note"
-	"github.com/SUSE/saptune/sap/solution"
-	"github.com/SUSE/saptune/txtparser"
+	"github.com/SUSE/saptune_v1/sap/note"
```

Comment:
@angelabriel To avoid changing all the go internals path to have these two binaries scenarios, that is tedious and error-prone, would it work if you change on the package build to use the option `go build -o` to generates the binary with the name saptune_v1? This way, maybe we can solve this almost only with packaging...

Reply:
removed version from internal go module path. The name 'saptune_v1' is generated during package build